src/languageClient.ts

```diff
@@ -63,6 +64,7 @@ export class LanguageClientManager implements LanguageClient {
   }
 
   async dispose (): Promise<void> {
```

Comment:
to be sure to understand: if this is disposed, it's not possible to call the start again, right?

Reply:
right!